2021/Robot2021/src/main/cpp/subsystems/Shooter.cpp

```diff
@@ -238,7 +241,7 @@ void Shooter::SetShooterSpeed(int level)
             m_targetOutput = frc::SmartDashboard::GetNumber("SH_SmartDashOutput", 0.0);
             break;
         default:
-            std::printf("2135: SH invalid velocity requested - %d\n", level);
+            spdlog::warn("2135: SH invalid velocity requested - {}", level);

```

Comment:
Remove the 2135 at the front of the statement

Reply:
Thank you, done!